runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -963,7 +956,7 @@ endfunc
 " let s:winbar_winids = []
 
 " Install the window toolbar in the current window.
-func s:InstallWinbar()
+func s:InstallWinbar(force)
```

Comment:
Missing check for config

Reply:
I don't understand, this function is a nop in Neovim -- one was or another?